### DISCUSSION TITLE
Google play 安卓报错

### DISCUSSION THREAD

**Discussion by timtang2010** [gkd-kit/gkd#720] [General]
刚刚看到出新版，Google play 报错
![Screenshot_20240830-102401.jpg](https://github.com/user-attachments/assets/448c7674-9e8a-414a-ad09-6aaee4aa2500)

杀毒软件也报错
![Screenshot_20240830-102144.jpg](https://github.com/user-attachments/assets/b263e7cc-62c4-47e8-b79c-bf072d91d9da)

**Comment by lisonge** [comment]
https://github.com/orgs/gkd-kit/discussions/614
